server.go

```diff
@@ -106,7 +106,7 @@ var errServerClosed = errors.New("server was closed")
 // If the poolLimit argument is greater than zero and the number of sockets in
 // use in this server is greater than the provided limit, errPoolLimit is
 // returned.
-func (server *mongoServer) AcquireSocket(poolLimit int, timeout time.Duration) (socket *mongoSocket, abended bool, err error) {
+func (server *mongoServer) AcquireSocket(poolLimit int, minPoolSize int, timeout time.Duration) (socket *mongoSocket, abended bool, err error) {
```

Comment:
I think it is worth updating the documentation with the new argument(s).  This is as I think about the fact that we should try to get these changes upstreamed and/or release our own fork as the upstream, since the library is now unmaintained.